src/Microsoft.Azure.ServiceBus/Core/MessageSender.cs

```diff
@@ -453,6 +458,11 @@ static int ValidateMessages(IList<Message> messageList)
                 throw Fx.Exception.ArgumentNull(nameof(messageList));
             }
 
+            if (messageList.Count == 0)
```

Comment:
>if (messageList.Count == 0) [](start = 12, length = 27)

This isnt required

Reply:
As in let `foreach` run with zero iterations? Could do that, but then it's less obvious what will happen, no?

Reply:
Done